### DISCUSSION TITLE
Sending parameters from server side

### DISCUSSION THREAD

**Discussion by MaTToX3** [Mimetis/Dotmim.Sync#534] [Q&A] [answered]
In our case we are setting some parameter on the server side (retrieving current user based on passed token), like so:

```
orchestrator.OnDatabaseChangesSelecting(dcsa =>
{
    var parameters = dcsa.Context.Parameters;

    if (!parameters.InnerCollection.Where(x => x.Name == "ID_User").Any())
    {
        parameters.Add("ID_User", userId);
    }
});
```

Since we don't have that user id on the client, I would like to send it there as well. Is this possible?

**Comment by Mimetis** [comment]
Maybe this https://github.com/Mimetis/Dotmim.Sync/discussions/391#discussioncomment-299575 can help you ?

**Reply by MaTToX3** [reply]
Still on version 0.5.7. Had to try a couple of interceptors to find the correct one. Thanks. :)

**Comment by MaTToX3** [comment, marked as answer]
It can be done like this:

```
_agent.LocalOrchestrator.OnDatabaseChangesApplied(async (args) => 
{
    var userId = args.Context.Parameters["ID_User"];
    ...
});
```
